### PR TITLE
Adding getLadder/getBlade functions for 80X

### PR DESCRIPTION
**Pull request by minano**
Cleaning hard code in SiPixelMonitorClient/SiPixelActionExecutor.\* adding some helper functions with no dependence on detector geometry for 80X

### CODE REVIEW

**Comment by deguio** [comment]
please test

File: DQM/SiPixelMonitorClient/src/SiPixelActionExecutor.cc [hunk 1]
```diff
@@ -202,7 +203,9 @@ void SiPixelActionExecutor::createSummary(DQMStore::IBooker & iBooker, DQMStore:
 //=============================================================================================================
 void SiPixelActionExecutor::bookDeviations(DQMStore::IBooker & iBooker, bool isUpgrade) {
   int nBPixModules;
+  int nFPixModules;
   if (isUpgrade) {nBPixModules=1184;} else {nBPixModules=768;} 
+  if (isUpgrade) {nFPixModules=672;} else {nFPixModules=672;}
```
Comment (deguio) [inline DQM/SiPixelMonitorClient/src/SiPixelActionExecutor.cc]: does this make sense?

File: DQM/SiPixelMonitorClient/src/SiPixelActionExecutor.cc [hunk 2]
```diff
@@ -213,17 +216,21 @@ void SiPixelActionExecutor::bookDeviations(DQMStore::IBooker & iBooker, bool isU
...
 }
 
 
-void SiPixelActionExecutor::fillDeviations(DQMStore::IGetter& iGetter) {
-  int n = 768;
+void SiPixelActionExecutor::fillDeviations(DQMStore::IGetter& iGetter, bool isUpgrade) {
+  int nBPixModules;
+  int nFPixModules;
+  if (isUpgrade) {nBPixModules=1184;} else {nBPixModules=768;} 
+  if (isUpgrade) {nFPixModules=672;} else {nFPixModules=672;}
```
Comment (deguio) [inline DQM/SiPixelMonitorClient/src/SiPixelActionExecutor.cc]: same here

**Comment by deguio** [comment]
-1
something needs to be fixed

File: DQM/SiPixelMonitorClient/src/SiPixelActionExecutor.cc (hunk 1 above)
Comment (davidlt) [inline DQM/SiPixelMonitorClient/src/SiPixelActionExecutor.cc]: It doesn't. It's just `nFPixModules=672`. Please don't use plain numbers like that (especially if they are used multiple times across code base). Add const static or pre-processor macro defining it and adding a comment line explaining what it means.

**Comment by davidlange6** [comment]
since its for 2017, save this PR for 81x (where dqm has already rejected at this point)

**Pull request closed without merging**